modules/book.js

```diff
@@ -0,0 +1,72 @@
+class Book {
+  books = [];
+
+  constructor(title, author) {
+    const randomId = Math.floor(Math.random() * 100000);
+    this.id = randomId;
+    this.title = title;
+    this.author = author;
+  }
+
+  storeBookLocally(bookArray) {     // eslint-disable-line
```

Comment:
- [ ] It will be great to use arrow functions here. This is the new style of functions in the es6 syntax. You can refactor this method to look as follows ⬇️ 

```
storeBookLocally = (bookArray) =>{   
    window.localStorage.setItem('bookArray', JSON.stringify(bookArray));
  }
```